build.sbt

```diff
@@ -76,6 +76,21 @@ lazy val coreJVM = core.jvm
 lazy val coreJS = core.js.settings(
   libraryDependencies += "io.github.cquiroz" %%% "scala-java-time" % "2.0.0-RC3" % Test
 )
+lazy val codegen = project
```

Comment:
I think `codegen` needs to be added to `root` above: `.aggregate(coreJVM, coreJS, http4s, akkaHttp, catsInteropJVM, catsInteropJS)`, otherwise it won't get published.